### PR TITLE
add slow-links-in-page-assets.md

### PR DESCRIPTION
**Pull request by garethnic**

### CODE REVIEW

File: content/links/slow-links-in-page-assets.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Having slow links to your page assets can severely affect the loading time of your pages. You will risk user engagement, declining search engine ranking and possibly reduced income.
+
+
+# How do I fix this ?
+
+* Use image sprites.
```
Comment (Johanndutoit) [inline content/links/slow-links-in-page-assets.md]: Think you're missing on of the biggies here. Use a CDN to better cache and speed up the downloads

Comment (Johanndutoit) [inline content/links/slow-links-in-page-assets.md]: ^ think that should be our main answer with the nice list you made listed for more options to look at

Comment (garethnic) [inline content/links/slow-links-in-page-assets.md]: Trust me to forget the obvious lol.

**Comment by Johanndutoit** [comment]
LGTM

**Pull request merged**